db/migrate/20190104134224_fix_oracle_constraints_references_to_index_names.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class FixOracleConstraintsReferencesToIndexNames < ActiveRecord::Migration
```

Comment:
Just in case that we need it someday (hopefully not), this is what I did originally:
```ruby
# frozen_string_literal: true

class FixOracleConstraintsReferencesToIndexNames < ActiveRecord::Migration
  def up
    return unless System::Database.oracle?

    db_connection = ActiveRecord::Base.connection
    query = <<~SQL
      SELECT uc.constraint_name, listagg(ac.column_name,', ') within group(order by ac.column_name) AS column_names,
        listagg(uc.table_name,', ') within group(order by uc.table_name) AS table_name
      FROM user_constraints uc JOIN all_cons_columns ac ON (uc.constraint_name = ac.constraint_name)
      WHERE uc.owner = 'RAILS' AND uc.constraint_type = 'U' AND uc.constraint_name != uc.index_name AND uc.table_name IN (
        SELECT table_name FROM all_tables WHERE owner = SYS_CONTEXT('userenv', 'current_schema') AND secondary = 'N'
      )
      GROUP BY uc.constraint_name
    SQL
    cursor = db_connection.execute(query)
    while (row = cursor.fetch_hash)
      constraint_name, table_name = row.values_at('CONSTRAINT_NAME', 'TABLE_NAME')
      db_connection.execute("ALTER TABLE #{table_name} DROP CONSTRAINT #{constraint_name}")
      db_connection.execute("ALTER TABLE #{table_name} ADD  CONSTRAINT #{constraint_name} UNIQUE (#{row['COLUMN_NAMES']}) USING INDEX #{constraint_name}")
    end
  end

  def down; end
end
```
But we decided that was risky and unnecessary because only these constraints in the list are affected.